posts/MichelaRocchettiBlogPost/Michela_JSoC.qmd

```diff
@@ -0,0 +1,43 @@
+---
+title: "JSoC'24 @JuliaHealth"
+description: "A resumee of my project work for Julia Summer of Code"
+author: "Michela Rocchetti"
+date: "9/26/2024"
+toc: true
+categories:
+  - news
+  - code
+  - analysis
+---
+# About me 
+Hi, I am Michela, I am from Rome and I hold a Master's degree in Physics of complex systems. My interest topics are the usage of modeling and AI methodologies in healthcare.
```

Comment:
Just a quick comment: feel free to link to your own website or blog if you have one, your LinkedIn, etc. in this description about yourself. I want this blog post to be like a great advertisement for you and your work so far. :smiley:

Reply:
Additionally, I think either in this "About Me" section or somewhere should be a note about how you found out about GSoC, what brought you to JuliaHealth, and what made you want to work on the IPUMS.jl project. Maybe separate section called, "Background"?